community/index.md

```diff
@@ -91,6 +91,9 @@ grubhub_user:
 contaazul_user:
   image_path: assets/images/users/contaazul_logo.svg
   alt: "ContaAzul Logo"
+xiaohongshu_user:
+  image_path: assets/images/users/xiaohongshu_logo.svg
```

Comment:
I believe this should be jpg, not svg.

Reply:
Also, the image file should be placed in assets/images/users/

Reply:
Ah I see. The image you added, @thomassun is a jpg, but you referenced it as an svg. @skim1420 is right, should be jpg.

If you have an svg version, that would be great. Or  a png would also be better than jpg. However jpg is acceptable.